### PR TITLE
Added project file Platform_Game.yml

### PR DESCRIPTION
**Pull request by EdwardDunn**
I have added a new project file called Platform-Game.yml. This is a new project designed for beginners. It is a simple JavaScript game using the HTML canvas feature.

### CODE REVIEW

**Comment by shiftkey** [comment]
@EdwardDunn could you add some issues to the repository, so that others can get involved?

**Comment by EdwardDunn** [comment]
@shiftkey Thanks for the advice, i have added some more issues to help people start contributing to the project.

**Comment by shiftkey** [comment]
@EdwardDunn these new issues are using the `help-wanted` label, rather than `up-for-grabs`. I'm not fussed on which one you prefer, but the link as-is goes to an empty list of issues.

**Comment by EdwardDunn** [comment]
@shiftkey Thanks i have added the up-for-grabs label to the issues.

File: _data/projects/Platform_Game.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Platform-Game
+desc: HTML/Javascript game, made using HTML canvas feature
+site: https://github.com/EdwardDunn/Platform-Game
+tags:
+- HTML
+- Javascript
+- CSS
+upforgrabs:
+  name: Help Wanted
```
Comment (shiftkey) [inline _data/projects/Platform_Game.yml]: This should be `up-for-grabs` now to match with the issue label

**Comment by EdwardDunn** [comment]
@shiftkey Sorry i should have realised that needed changing as well, i have updated the project file.

**Pull request merged**